xml/tuning_cgroups.xml

```diff
@@ -58,58 +58,33 @@
   <itemizedlist>
    <listitem>
     <para>
-     hybrid: v2 hierarchy without controllers, and the controllers are on v1
-     hierarchies
+     unified: v2 hierarchy with controllers
     </para>
    </listitem>
    <listitem>
     <para>
-     unified: v2 hierarchy with controllers
+     hybrid: v2 hierarchy without controllers, and the controllers are on v1
+     hierarchies (deprecated)
     </para>
    </listitem>
   </itemizedlist>
   <para>
-   The current default mode is hybrid. This
-   provides backwards compatibility for applications that need it.
-   The following features are available only with the unified v2 hierarchy:
+   The default mode is unified.
+   There is a hybrid mode that provides backwards compatibility for applications
+   that need it.
   </para>
-  <itemizedlist>
-   <listitem>
-    <para>
-     memory controller: reclaim protection (aka memory.low), memory.high,
-     PSI (pressure stall information)
-    </para>
-   </listitem>
-   <listitem>
-    <para>
-    io controller: writeback control, new control policies
-   </para>
-  </listitem>
-  <listitem>
-   <para>
-    controller delegation to non-privileged users (rootless containers)
-   </para>
-  </listitem>
-  <listitem>
-   <para>
-    freezer support in systemd
-   </para>
-  </listitem>
-  <listitem>
-   <para>
-    simpler handling of the single hierarchy
-   </para>
-  </listitem>
- </itemizedlist>
    <para>
     You may set only one mode.
   </para>
-   <para>
-   To enable the unified control group hierarchy, append
-   <option>systemd.unified_cgroup_hierarchy=1</option> as a kernel command
-   line parameter to the &grub; boot loader. For more details about configuring &grub;, refer to
-   <xref linkend="cha-grub2"/>.
-  </para>
+  <sect3 xml:id="sec-cgroups-hybrid-hierarchy">
+  <title>Hybrid cgroup hierarchy (cgroup v1, deprecated)</title>
```

Comment:
Assuming that visibility is the goal here, I recommend providing the deprecation information as a `<note>...</note>` admonition, instead of including it in the title text. 
For more information, refer to our style guide: https://documentation.suse.com/style/current/single-html/docu_styleguide/#sec-admonition

Reply:
This proposal seems fine. Thanks.